rewrite-gradle/src/main/java/org/openrewrite/gradle/UpgradeDependencyVersion.java

```diff
@@ -300,6 +300,16 @@ public J visitMethodInvocation(J.MethodInvocation method, ExecutionContext ctx)
     @Override
     public TreeVisitor<?, ExecutionContext> getVisitor(DependencyVersionState acc) {
         return new TreeVisitor<Tree, ExecutionContext>() {
+            @Override
+            public boolean isAcceptable(SourceFile sf, ExecutionContext ctx) {
+                if (sf instanceof Properties) {
+                    return new UpdateProperties(acc).isAcceptable(sf, ctx);
+                } else if (sf instanceof G.CompilationUnit) {
+                    return new UpdateGroovy(acc).isAcceptable(sf, ctx);
+                }
+                return false;
```

Comment:
```suggestion
                return new UpdateProperties(acc).isAcceptable(sf, ctx) || new UpdateGroovy(acc).isAcceptable(sf, ctx);
```

We should also use `isAcceptable()` rather than `instanceof` checks in the `visit()` override, since hypothetically those recipes could further customize `isAcceptable()`

Reply:
Hope I understood that correctly; Applied in fcf6032cd0a1871207b2f128fddde7a4506c9609